src/pytorch_lightning/loggers/mlflow.py

```diff
@@ -37,7 +37,7 @@
 
 log = logging.getLogger(__name__)
 LOCAL_FILE_URI_PREFIX = "file:"
-_MLFLOW_AVAILABLE = RequirementCache("mlflow>=1.0.0")
+_MLFLOW_AVAILABLE = RequirementCache("mlflow>=1.0.0") or RequirementCache("mlflow-skinny>=1.0.0")
```

Comment:
Could you also mention this alternative in the docstring below, where `pip install mlflow` is mentioned?